### PR TITLE
.travis.yml: Modify CI for Jekyll

### PR DESCRIPTION
**Pull request by hemangsk**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-ignore = vendors/**, _site/**
+ignore = resources/vendors/**, _site/**
 use_spaces = True
 
 [html]
 bears = HTMLLintBear
 files = **.html
 htmllint_ignore = optional_tag, protocol, indentation,  concerns_separation, capitalization
-ignore = vendore/**, _site/**
+ignore = resources/vendore/**, _site/**
```
Comment (nkprince007) [inline .coafile]: `vendore` ?

File: index.html [hunk 2]
```diff
@@ -51,7 +51,7 @@
   </main>
   <footer class="page-footer footer-light transparent">
     <div class="row">
-      <div class="col m6 offset-m3 center"> <img class="footer-logo" src="https://api.coala.io/en/latest/_static/images/coala_logo.svg"> <span class="footer-title">{{ site.organization }}</span> </div>
+      <div class="col m6 offset-m3 center"> <img class="footer-logo" src="https://api.coala.io/en/latest/_static/images/coala_logo.svg" alt="coala"> <span class="footer-title">{{ site.organization }}</span> </div>
```
Comment (nkprince007) [inline index.html]: This change should be in the other commit, right? https://github.com/coala/projects/pull/333/commits/c579dc90bdcc25ed68ca30b5e36452725e65ce46

Comment (hemangsk) [inline index.html]: Yep, thanks! moved it to that commit

**Comment by nkprince007** [comment]
ack 
https://github.com/coala/projects/pull/333/commits/e274a85451985285167ba2f7e8ad3bdf6e4a13f
https://github.com/coala/projects/pull/333/commits/09285908f88e7c72b5f62ae7723e1cba8e62006
https://github.com/coala/projects/pull/333/commits/7ea691b5a03eb9c4d4a3c23038b0ce99a18fcf27

**Comment by hemangsk** [comment]
@rultor merge

File: README.md [hunk 3]
```diff
@@ -28,16 +28,18 @@ Why?
 ## Usage
 
 To clone the repository and run this website on your local machine, [install Jekyll](https://jekyllrb.com/docs/installation/) for your OS and type the following commands:
-
+
```
Comment (Arsfiqball) [inline README.md]: trailing whitespace :/

Comment (hemangsk) [inline README.md]: Why is it trailing? that's for a line break right?

**Pull request merged**